### ISSUE TITLE
Requirement of training script.

### ISSUE THREAD

**Issue by manask0412** [X-Men01/YOLOv10-Fire-and-Smoke-Detection#2]
Hii @X-Men01 

Great Work !!
Can you please provide the training script used for training this model ??
I would love to get some more knowledge by understanding the model training pipeline..

**Comment by tjyge** [comment]
Hii @X-Men01

Amazing work on this project! I am also interested in the training script used for training this model.

I’ve been retraining the model using the following setup:

model = YOLO("yolov11m.pt")
model.train(
    data="data.yaml",
    epochs=20,
    imgsz=960,
    batch=16,
    optimizer="AdamW",
    lr0=0.001,
    weight_decay=0.01
)

I noticed from your results that you trained for around 400 epochs, which is quite impressive!
Could you please share a bit more about your training pipeline — such as:

1. The GPU you used.
2. Approximately how long the training took.
3. Any additional settings or augmentations you applied?

I’d love to understand your setup better and see how I can align my training process to achieve similar accuracy.
Thanks a lot for your time and for sharing such great work! 🙏
